src/web/components/Core/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { ReactNode } from 'react';
+
+import './Accordion.scss';
+
+export type AccordionProps = {
+  title: string;
+  content: ReactNode;
+  defaultExpanded: boolean;
+  isCollapsible: boolean;
+  label: ReactNode;
+};
+
+export function Accordion({
+  title,
+  content,
+  defaultExpanded,
+  isCollapsible,
+  label,
+}: AccordionProps) {
+  const itemName = 'item-1';
```

Comment:
the defaultValue on the Root and the value of the item need to be the same for it to show by default. let me know if you can think of a better way.